src/ray/gcs/gcs_server/gcs_node_manager.cc

```diff
@@ -317,7 +317,7 @@ void GcsNodeManager::UpdateNodeRealtimeResources(
   if (!light_report_resource_usage_enabled_ ||
       gcs_resource_manager_->GetClusterResources().count(node_id) == 0 ||
       resource_data.resources_available_changed()) {
-    gcs_resource_manager_->UpdateResources(
+    gcs_resource_manager_->SetAvailableResources(
```

Comment:
I'm not sure if there are new resources updated here, if so, the total resources should be updated too.

Reply:
The total resources is updated by `HandleUpdateResources`.